### ISSUE TITLE
Let us decide the index of the blank in tf.nn.ctc_greedy_decoder 

### ISSUE THREAD

**Issue by FLming** [tensorflow/tensorflow#32903]
**System information**
- TensorFlow version (you are using): 2.0.0rc1 
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

I am using ctc_greedy_decoder function to decode my output from logits. In the ctc_loss function, we have the opportunity to choose blank_index equals what, but in the ctc_greedy_decoder function, it seems equals (num_classes-1) by default. Thus I have to write a map function by myself or change my lookup table

**Will this change the current api? How?**

Please add a parameter to this function in order to choose which index we use.

**Comment by pvarouktsis** [comment]
Hello! Is anyone working on this? If not could I have a try?

**Comment by ebrevdo** [comment]
Yes; please feel free.

**Comment by pvarouktsis** [comment]
Thank you!

**Comment by FLming** [comment]
Although I really want to do this, I don’t know how to contribute it. I have gone to view the implementation of this function, but I can't find the gen_ctc_ops module.

**Comment by ebrevdo** [comment]
I believe it's in core/ops/ctc_ops.cc.

**Comment by tilakrayal** [comment]
@FLming ,
Related PR was merged.Can you please let us confirm if the issue has resolved with the merged PR.

**Comment by FLming** [comment]
OK, I will check it in this week.

**Comment by FLming** [comment]
LGFM. Thanks.
